src/components/CardView/CardView.js

```diff
@@ -1,11 +1,17 @@
 import React, { PropTypes } from 'react';
 
 const CardView = (props) => {
-  const { title, text, imageUrl, imageAlt, buttonUrl, buttonText } = props;
+  const { title, text, imageUrl, imageAlt, imageWidth, imageHeight, buttonUrl, buttonText } = props;
 
   let imageEle;
```

Comment:
Changed mind already :D Updating to the multiline ternary, the indentation for it make it clear enough still.